charts/thoras/templates/api-server-v2/deployment.yaml

```diff
@@ -32,6 +32,27 @@ spec:
         - name: tls
           secret:
             secretName: thoras-api-server-cert
+      {{- if .Values.metricsCollector.timescale.enabled }}
```

Comment:
It feels kind icky referring to the metrics collector settings from the api service layer deployment. Open to feedback here.